ci/copy-unix-release-artifacts.sh

```diff
@@ -104,4 +104,8 @@ if [[ "$NAME" == "linux" ]]; then
     cp bazel-bin/docs/sphinx-source-tree.tar.gz $OUTPUT_DIR/split-release/docs/sphinx-source-tree-$RELEASE_TAG.tar.gz
     cp bazel-bin/docs/pdf-fonts-tar.tar.gz $OUTPUT_DIR/split-release/docs/pdf-fonts-$RELEASE_TAG.tar.gz
     cp bazel-bin/docs/non-sphinx-html-docs.tar.gz $OUTPUT_DIR/split-release/docs/non-sphinx-html-docs-$RELEASE_TAG.tar.gz
+
+    bazel build //ledger/test-evidence:generate-security-test-evidence-files
+    cp bazel-bin/ledger/test-evidence/security-test-evidence.csv $OUTPUT_DIR/github/security-test-evidence.csv
```

Comment:
consider to prefix the output files with `daml-` such that we can easily differentiate them from the canton ones when processing them in the assembly repo later.

Reply:
sure, will do so